src/2week/src/utils/array.ts

```diff
@@ -0,0 +1 @@
+export const push = <T = any>(arr: T[], item: T) => [...arr, item];
```

Comment:
혹시 any보다는 unknown을 사용하는 건 어떨까요?
any와 동일하게 상위 타입을 가리키지만 unknown은 어떤 타입이 들어갈 지 모르기에 any처럼 무분별한 사용이 되지는 않으니 더 안전하게 사용할 수 있을 것 같습니다!
```suggestion
export const push = <T = unknown>(arr: T[], item: T) => [...arr, item];
```